node/devnet/src/lib.rs

```diff
@@ -2,15 +2,15 @@
 use std::sync::Arc;
 use std::net::{IpAddr, Ipv4Addr, SocketAddr};
 
-use futures::sync::mpsc::{channel, Sender};
+use futures::sync::mpsc::{channel, Receiver};
 use futures::future;
+use futures::{Stream, Future};
 
 use configs::{get_devnet_configs, ClientConfig, DevNetConfig, RPCConfig};
 use client::Client;
-use consensus::adapters::transaction_to_payload;
 use consensus::passthrough::spawn_consensus;
-use primitives::transaction::SignedTransaction;
-use primitives::chain::ChainPayload;
+use primitives::chain::{ReceiptBlock};
```

Comment:
nit: no need {}

also generally ordering of imports (e.g. std, external crates, ours, current crate) - ctrl+option+O in clion to auto order.